xt/01.2-deps.t

```diff
@@ -34,8 +34,10 @@ find(\&collect, 'lib/', 'old/bin/', 'old/lib/');
 
 push @on_disk, 'tools/starman.psgi';
 
+#TODO: Implement Template plugins <?lsmb USE plugin ?>
 ok_dependencies($file, \@on_disk,
                 phases => 'runtime',
                 ignores => [ 'Image::Size', 'LaTeX::Driver',
+                             'Template::Plugin::JSON',
```

Comment:
If Template::Plugin::JSON isn't used anywhere, it would be good to document in `cpanfile` why it's required. (Is it still?)

Reply:
It is used by new_user.html but we do not currently recognize Template plugins in the dependencies.